src/app/process/process.component.html

```diff
@@ -42,7 +42,7 @@ <h1 tabindex="0">Forest Landscape Planning</h1>
           </p>
           <p>
             For more information on forest landscape planning, please visit the B.C. government’s
-             <a href="https://www2.qa.gov.bc.ca/gov/content?id=A6A208C8A4224EA1BEEC38B337DDA1B8">forest landscape planning homepage.</a>
+             <a href="https://www2.gov.bc.ca/gov/content?id=A6A208C8A4224EA1BEEC38B337DDA1B8">forest landscape planning homepage.</a>
```

Comment:
Corresponds with the permalink on https://www2.gov.bc.ca/gov/content/industry/forestry/managing-our-forest-resources/forest-landscape-plans 👍

Reply:
Observant of you to notice!